docs/SETUP/FRONT_END.md

```diff
@@ -20,6 +20,10 @@ If you are running Windows, you will need to [install NVM for Windows](https://g
 
 Note: make sure to keep your fork up to date with the original repository by following the instructions [here](https://docs.github.com/en/get-started/quickstart/fork-a-repo#keep-your-fork-synced).
 
+### Precommit hook
+
+We use a precommit hook to help with formatting and linting in our CI/CD pipeline. When setting up the repo, please first [make sure you have `pre-commit` installed](https://pre-commit.com/) using `pip` or another package manager. Once that's done, run `pre-commit install` in the root directory to set up the precommit hooks (configured in `.pre-commit-config.yaml`).
```

Comment:
Does `pre-commit` require installing Python and pip? Or can `pre-commit` be installed with NPM?

If it requires Python and pip, then this adds an unnecessary hurdle for front-end developers. Right now, these contributors can run and develop the React front-end without ever using Python or touching the backend data pipeline. It seems arbitrary to make them install Python and pip.

Reply:
@CodeWritingCow heard, but my understanding is that Python 3 ships with Unix systems, and [`pip` is normally bundled with it](https://pip.pypa.io/en/stable/installation/). So unless a contributor is only running Windows, they should actually have these already installed on their machine. We can link to installation instructions for Python for Windows for those cases.